tests/widgets_list.rs

```diff
@@ -20,7 +20,7 @@ fn list_should_shows_the_length() {
     assert_eq!(list.len(), 3);
     assert!(!list.is_empty());
 
-    let empty_list = List::new(vec![]);
+    let empty_list = List::new::<Vec<&str>>(vec![]);
```

Comment:
nit: The `::<Vec<&str>>` part looks a bit odd. Does something like this look better?
```suggestion
    let empty_list = List::new(iter::empty::<ListItem>());
```
or `vec::new::<ListItem>()` perhaps?

Reply:
Or even better, we can now use the Default implementation to create an empty list. I should add documentation for that